resources/compass/charts/kyma-environment-broker/values.yaml

```diff
@@ -51,7 +51,7 @@ additionalRuntimeComponents:
   - name: "knative-eventing-kafka"
     namespace: "knative-eventing"
 
-kymaVersion: "1.11.0-rc2"
+kymaVersion: "master-347cca4d"
```

Comment:
To use different kyma version you can pass it as a parameter during provisioning. Changing verison in chart is not neccessary.